ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/title/v1/title/_cq_dialog/.content.xml

```diff
@@ -18,9 +18,9 @@
     jcr:primaryType="nt:unstructured"
     jcr:title="Title"
     sling:resourceType="cq/gui/components/authoring/dialog"
-    extraClientlibs="[core.wcm.components.title.v2.editor]"
+    extraClientlibs="[core.forms.components.title.v1.editor]"
```

Comment:
there is no such library `core.forms.components.title.v1.editor` in the codebase.
ideally, this should come from wcm text inheritance implicitly via resourcesupertype. can you pls remove this and see if https://github.com/adobe/aem-core-wcm-components/blob/a7e377c6810dd2125a902d3ef3df5c92ec2a1719/content/src/content/jcr_root/apps/core/wcm/components/title/v2/title/clientlibs/editor/js/title.js is getting loaded in the browser while opening edit of text or not.